### PR TITLE
Fix typos in `eventregistration` & blog post

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR fixes some typos discovered by @DerVogel2020 in https://github.com/corona-warn-app/cwa-website/pull/1132#pullrequestreview-643532898.
This PR also changes "Corona warning app" to "Corona-Warn-App" in the English blog post "2021-04-23-qr-code-generator-website".

### CODE REVIEW

File: blog/2021-04-23-qr-code-generator-website/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The Corona-Warn-App community has developed a QR code generator for the app's we
 
 <!-- overview -->
 
-The Corona-Warn-App community provided the QR code generator to the project team on GitHub. The development team then reviewed the community's contribution and released the QR code generator in a customized form. We would like to take this opportunity to thank in particular @bastianjoel and all the community members involved for contributing to this new feature. This example shows again very well how useful the open source approach and the publication of the source code of the Corona warning app is.
+The Corona-Warn-App community provided the QR code generator to the project team on GitHub. The development team then reviewed the community's contribution and released the QR code generator in a customized form. We would like to take this opportunity to thank in particular @bastianjoel and all the community members involved for contributing to this new feature. This example shows again very well how useful the open source approach and the publication of the source code of the Corona-Warn-Ap is.
```
Comment (MikeMcC399) [inline blog/2021-04-23-qr-code-generator-website/index.md:14]: Corona-Warn-Ap => Corona-Warn-App !

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There is a "p" missing in the word "App".

**Comment by Ein-Tim** [comment]
Thanks!

**Review by DerVogel2020 — APPROVED** [review]

**Comment by heinezen** [comment]
LGTM

**Review by heinezen — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Many thanks.

**Pull request merged**